manifests/4.4/cluster-kube-descheduler-operator.v4.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,233 @@
+        "RemoveDuplicates":
+           enabled: true
+        "LowNodeUtilization":
+           enabled: true
+           params:
+             nodeResourceUtilizationThresholds:
+               thresholds:
+                 "cpu" : 10
+                 "memory": 20
+                 "pods": 30
+               targetThresholds:
+                 "cpu" : 40
+                 "memory": 50
+                 "pods": 60
+               numberOfNodes: 3
+      ```
+
+      The above configmap would be mounted as a volume in descheduler pod created. Whenever we change strategies, parameters or schedule in the CR, the descheduler operator is responsible for identifying those changes and regenerating the configmap. For more information on how descheduler works, please visit [descheduler](https://docs.openshift.com/container-platform/3.11/admin_guide/scheduling/descheduler.html)
+
+
+      ## Parameters
+      The Descheduler operator exposes the following parameters in its CRD:
+
+      * `deschedulingIntervalSeconds` - this sets the number of seconds between descheduler runs
+      * `image` - specifies the Descheduler container image to deploy
+      * `flags` - this allows additional descheduler flags to be set, and they will be appended to the descheduler pod. Therefore, they must be in the same format as would be passed to the descheduler binary (eg, `"--dry-run"`)
+  displayName: Descheduler Operator
+  keywords: ["scheduling", "descheduler", "workload"]
+  provider:
+    name: Red Hat, Inc.
+  maturity: alpha # or beta/stable maybe?
+  version: 4.4.0
+  links:
+  - name: Source Code
+    url: https://github.com/openshift/cluster-kube-descheduler-operator
+  maintainers:
+  - email: support@redhat.com
+    name: Red Hat
+  minKubeVersion: v1.16.0 # or lower?
+  labels:
+    olm-owner-enterprise-app: cluster-kube-descheduler-operator
+    olm-status-descriptors: cluster-kube-descheduler-operator.v4.4.0
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  install:
+    spec:
+      clusterPermissions:
+      - serviceAccountName: openshift-descheduler
+        rules:
+        - apiGroups:
+          - operator.openshift.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - kubedeschedulers.operator.openshift.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          - pods
+          - configmaps
+          - secrets
+          - names
+          - nodes
+          - pods/eviction
+          - events
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - replicasets
+          verbs:
+          - "*"
+        - apiGroups: ["batch", "extensions"]
```

Comment:
What we use extensions for? We should not.

Reply:
We don't use cron jobs anymore after merging https://github.com/openshift/cluster-kube-descheduler-operator/pull/66/commits/b64e7c0cdb499fb52e02ed51e92f2e9d55e1ccb5?file-filters%5B%5D=.go#diff-ad0243c09cf8e6c22f5f54bd27636f36L235 so we might remove entire rule.